docs/api.md

```diff
@@ -173,6 +173,10 @@ The sub-fields are:
   * *numberOfPhysicalDisks* -- Integer, number of physical disks to use for the
     logical disk. Defaults to minimum number of disks required for the
     particular RAID level.
+  * *physicalDisks* -- List of names of physical disks (Strings). This is an
+    optional field. If specified, the `controller` field must be specified too.
```

Comment:
I was looking for a validation rule, either in the webhook or controller code, but didn't find it. @demonCoder95 shouldn't be required, as per documentation?

Reply:
Could you elaborate on the scope of this validation? What do we need to validate?

Reply:
That the `controller` field is set when `physicalDisks` is specified

Reply:
Oh right, I've added it here. let me send the patch.

Reply:
All done. @andfasano